index.js

```diff
@@ -105,7 +162,19 @@ function findById() {}
  *  filterByGenre(movies, "Horror")
  *  //> []
  */
-function filterByGenre() {}
+function filterByGenre(movies, genre) {
+  let moviesByGenre = [];
+
+  for (let i = 0; i < movies.length; i ++){
+    let lowerCaseGenre = (movies[i].genre).toLowerCase();
+    
+    if ((lowerCaseGenre).includes(genre.toLowerCase())){
+      moviesByGenre.push(movies[i]);
+    }
```

Comment:
I really like using `toLowerCae()`on genre and `(movies[i].genre)`.  This is a great habit for sanitizing text that is input to a function.  Would this work with .`toUpperCase()`?